docs/source/examples.daisy_chain_gap.ipynb

```diff
@@ -0,0 +1,746 @@
+     ]
+    }
+   ],
+   "source": [
+    "\n",
+    "gap_params = {\n",
+    "    \"gap_file\": gap_fname,\n",
+    "    \"energy_parameter_name\": \"xtb_energy\", \n",
+    "    \"force_parameter_name\": \"xtb_forces\", \n",
+    "    \"default_sigma\": [0.001, 0.01, 0.0, 0.0], \n",
+    "    \"config_type_kernel_regularisation\": {\"isolated_atom\":[0.0001,0.0001,0.0,0.0]},\n",
+    "    \"_gap\": [{\n",
+    "            \"soap\": True,\n",
+    "            \"l_max\": 3,\n",
+    "            \"n_max\": 6, \n",
+    "            \"cutoff\": 3,\n",
+    "            \"delta\": 0.1,\n",
+    "            \"covariance_type\": \"dot_product\",\n",
+    "            \"zeta\": 4, \n",
+    "            \"n_sparse\":20, \n",
+    "            \"sparse_method\": \"cur_points\", \n",
+    "            \"atom_gaussian_width\":0.3,\n",
+    "            \"cutoff_transition_width\": 0.5},\n",
+    "         {\n",
+    "            \"soap\": True,\n",
+    "            \"l_max\": 3,\n",
+    "            \"n_max\": 6, \n",
+    "            \"cutoff\": 6,\n",
+    "            \"delta\": 0.1,\n",
+    "            \"covariance_type\": \"dot_product\",\n",
+    "            \"zeta\": 4, \n",
+    "            \"n_sparse\":20, \n",
+    "            \"sparse_method\": \"cur_points\", \n",
+    "            \"atom_gaussian_width\":0.6,\n",
+    "            \"cutoff_transition_width\": 1},\n",
+    "        {\n",
+    "            \"distance_2b\": True,\n",
+    "            \"cutoff\": 7, \n",
+    "            \"covariance_type\": \"ard_se\",\n",
+    "            \"delta\": 1,\n",
+    "            \"theta_uniform\": 1.0,\n",
+    "            \"sparse_method\": \"uniform\", \n",
+    "            \"n_sparse\": 10 \n",
+    "        }\n",
+    "    ]\n",
+    "}\n",
+    "\n",
+    "remote_info = {\n",
+    "    \"sys_name\" : \"local\", \n",
+    "    \"job_name\" : \"gap-fit\", \n",
+    "    \"resources\" : { \n",
+    "        \"max_time\" : \"15m\",\n",
+    "        \"num_cores\" : 4,\n",
+    "        \"partitions\" : \"any$\"}, \n",
+    "    \"partial_node\": True,\n",
+    "    \"check_interval\": 5, \n",
+    "    \"pre_cmds\" :[\"conda activate dev\"],\n",
+    "}\n",
+    "\n",
+    "# set to None for github testing purposes\n",
+    "remote_info = None\n",
+    "\n",
+    "if not Path(gap_fname).exists(): \n",
```

Comment:
Is this really not auto tested for skipping?

Reply:
No, but I think it should be. Have added `skip_if_present=False` to `run_gap_fit`, similarly to `run_ace_fit`. Or should it default to True?

Reply:
Did I really not already have that?  Was it in the multistep GAP, but just not in simple?

Reply:
I also re-added `OutputSpec(... set_tags=dict)` in 5445057423f